rules/best-practices.js

```diff
@@ -0,0 +1,64 @@
+// Best Practices http://eslint.org/docs/rules/#best-practices
+
+module.exports = {
+	"rules": {
+		"accessor-pairs": [ 2, { "setWithoutGet": true } ], // enforces getter/setter pairs in objects
+		"block-scoped-var": 2, // JSHINT treat var statements as if they were block scoped
+		"complexity": [ 2, 50 ], // specify the maximum cyclomatic complexity allowed in a program
```

Comment:
Personally I think this should be turned on, but I'm not sure what value we should set our default to be. *NOTE: 50 is way too high"

http://eslint.org/docs/rules/complexity.html